core/utils.py

```diff
@@ -362,7 +362,13 @@ def get_project_path(filepath):
         import os
         dir_path = os.path.dirname(filepath)
         if get_command_result("git rev-parse --is-inside-work-tree", dir_path) == "true":
-            return get_command_result("git rev-parse --show-toplevel", dir_path)
+            path_from_git = get_command_result("git rev-parse --show-toplevel", dir_path)
+            if get_os_name() == "windows":
```

Comment:
This line should change to:

if get_os_name() == "windows" and path_from_git.startswith("/"):

Reply:
Or only run convert code when env is MSYS2